features/timeseries/src/main/java/org/opennms/netmgt/timeseries/util/Utils.java

```diff
@@ -43,4 +46,18 @@
         }
         return map;
     }
+
+    public static boolean equals(final FetchResults results1, final FetchResults results2) {
```

Comment:
Why implement this as a seperate free-standing function and not as part of the class it's comparing?

Reply:
Hm good question. The method was also not used anymore. I guess some leftover from refactoring... I ended up deleting the whole class :-)